x-pack/plugins/maps/public/classes/styles/vector/properties/__tests__/test_util.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// eslint-disable-next-line max-classes-per-file
+import { FIELD_ORIGIN } from '../../../../../../common/constants';
+import { StyleMeta } from '../../style_meta';
+import {
+  CategoryFieldMeta,
+  GeometryTypes,
+  RangeFieldMeta,
+  StyleMetaDescriptor,
+} from '../../../../../../common/descriptor_types';
+import { IField } from '../../../../fields/field';
+import { AbstractVectorSource, IVectorSource } from '../../../../sources/vector_source';
+import { ITooltipProperty, TooltipProperty } from '../../../../tooltips/tooltip_property';
+
+export const mockField: IField = {
```

Comment:
I would recommend extending `AbstractField` to avoid writing a whole new IField implementation just for testing.

Here is an example https://github.com/elastic/kibana/blob/master/x-pack/plugins/maps/public/classes/tooltips/es_tooltip_property.test.ts#L13